modules/bs.environment/data/bs.environment/function/celestial/look_at_sun.mcfunction

```diff
@@ -0,0 +1,20 @@
+# ------------------------------------------------------------------------------------------------------------
+# Copyright (c) 2025 Gunivers
+#
+# This file is part of the Bookshelf project (https://github.com/mcbookshelf/bookshelf).
+#
+# This source code is subject to the terms of the Mozilla Public License, v. 2.0.
+# If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Conditions:
+# - You may use this file in compliance with the MPL v2.0
+# - Any modifications must be documented and disclosed under the same license
+#
+# For more details, refer to the MPL v2.0.
+# ------------------------------------------------------------------------------------------------------------
+
+execute store result storage bs:ctx y float 0.001 run function bs.environment:celestial/get_sun_angle
+execute store result score #t bs.ctx run time query daytime
+execute if score #t bs.ctx matches 6000..18000 run data modify storage bs:ctx x set value 90
```

Comment:
Since it's only 2 possible states i think it's better in this case to have 2 rotate function and a single macro param.
Then we have a single if score with a return run function and the other rotate function just after